discord_bots/commands.py

```diff
@@ -2127,6 +2127,19 @@ async def disableleaderboard(ctx: Context):
     )
 
 
+@bot.command()
+async def disablestats(ctx: Context):
+    session = Session()
+    player = session.query(Player).filter(Player.id == ctx.message.author.id).first()
+    player.stats_enabled = False
+    session.commit()
+    await send_message(
+        ctx.message.channel,
+        embed_description="!stats disabled",
+        colour=Colour.blue()
+    )
```

Comment:
```suggestion
    )
    session.close()
```
this releases the session faster on the db